libyul/optimiser/ASTCopier.cpp

```diff
@@ -47,10 +47,12 @@ Statement ASTCopier::operator()(VariableDeclaration const& _varDecl)
 
 Statement ASTCopier::operator()(Assignment const& _assignment)
 {
+	// throws and causes memleak in translateVector call, changed order to throw before allocation
```

Comment:
Add FIXME, TODO or @todo here?

Reply:
this is already fixed by changing the order as the comment says

Reply:
comment is maybe a bit ambiguous

Reply:
Yes but in cases where such order is required to maintain working code it has to be highlighted.

Reply:
You mean more highlighted than with the given comment?

Reply:
The comment is not very clear to me tbh.